### DISCUSSION TITLE
How to damage the player with setHealth? (1.5 MODCHART)

### DISCUSSION THREAD

**Discussion by PlayerPro342** [KadeArchive/Kade-Engine#706] [Q&A] [answered]
I tried -0.1, true, it killed me.
How i damage and not kill instantly?
Do it need to be like -0.4 or more?

**Comment by PlayerPro342** [comment, marked as answer]
Im dumb

**Reply by puyoxyz** [reply]
setHealth *sets* the health not *changes* it

**Reply by PlayerPro342** [reply]
then like health - 0.2 ?

**Reply by puyoxyz** [reply]
`setHealth(getProperty("health") - 0.2)`

**Reply by PlayerPro342** [reply]
i will see

**Reply by PlayerPro342** [reply]
i think it worked thanks

**Reply by PlayerPro342** [reply]
yes

**Reply by PlayerPro342** [reply]
idea: player2 sing, curstep = <curStep> to <curStep> (noteData,pos)
if noteData = 0 then
      setHealth(getProperty("health") - 0.2)
end
if noteData = 1 then
      setHealth(getProperty("health") - 0.2)
end
if noteData = 2 then
      setHealth(getProperty("health") - 0.2)
end
if noteData = 3 then
      setHealth(getProperty("health") - 0.2)
end

**Reply by PlayerPro342** [reply]
i tried < and > it invis lol
